package.json

```diff
@@ -56,47 +56,47 @@
   },
   "devDependencies": {
     "@eslint/js": "^9.16.0",
-    "@microsoft/api-extractor": "^7.47.11",
-    "@revenuecat/purchases-ui-js": "0.0.14",
-    "@storybook/addon-essentials": "^8.3.6",
-    "@storybook/addon-interactions": "^8.3.6",
-    "@storybook/addon-links": "^8.3.6",
-    "@storybook/addon-svelte-csf": "^5.0.0-next.11",
-    "@storybook/blocks": "^8.3.6",
-    "@storybook/svelte": "^8.3.6",
-    "@storybook/sveltekit": "^8.3.6",
-    "@storybook/test": "^8.3.6",
+    "@microsoft/api-extractor": "^7.48.0",
+    "@revenuecat/purchases-ui-js": "0.0.16",
+    "@storybook/addon-essentials": "^8.4.7",
+    "@storybook/addon-interactions": "^8.4.7",
+    "@storybook/addon-links": "^8.4.7",
+    "@storybook/addon-svelte-csf": "^5.0.0-next.9",
+    "@storybook/blocks": "^8.4.7",
+    "@storybook/svelte": "^8.4.7",
+    "@storybook/sveltekit": "^8.4.7",
+    "@storybook/test": "^8.4.7",
     "@stripe/stripe-js": "^4.9.0",
-    "@sveltejs/vite-plugin-svelte": "^4.0.0",
+    "@sveltejs/vite-plugin-svelte": "^5.0.1",
     "@tsconfig/svelte": "^5.0.4",
-    "@typescript-eslint/eslint-plugin": "^6.10.0",
+    "@typescript-eslint/eslint-plugin": "^6.21.0",
     "eslint": "^8.57.1",
-    "eslint-config-prettier": "^9.0.0",
+    "eslint-config-prettier": "^9.1.0",
     "eslint-config-standard-with-typescript": "^43.0.1",
-    "eslint-plugin-import": "^2.29.0",
-    "eslint-plugin-n": "^16.2.0",
-    "eslint-plugin-prettier": "^5.0.1",
-    "eslint-plugin-promise": "^6.1.1",
-    "eslint-plugin-storybook": "^0.9.0",
+    "eslint-plugin-import": "^2.31.0",
+    "eslint-plugin-n": "^16.6.2",
+    "eslint-plugin-prettier": "^5.2.1",
+    "eslint-plugin-promise": "^6.6.0",
+    "eslint-plugin-storybook": "^0.11.1",
     "eslint-plugin-svelte": "^2.46.1",
     "globals": "^15.13.0",
     "http-status-codes": "^2.3.0",
     "husky": "^9.1.7",
-    "jsdom": "^22.1.0",
+    "jsdom": "^25.0.1",
     "lint-staged": "^15.2.10",
     "msw": "^2.0.4",
-    "prettier": "^3.4.1",
+    "prettier": "^3.4.2",
     "prettier-plugin-svelte": "^3.3.2",
-    "react": "^18.2.0",
-    "react-dom": "^18.2.0",
-    "storybook": "^8.3.6",
-    "svelte": "^5.5.0",
-    "svelte-check": "^4.0.6",
-    "svelte-stripe": "^1.1.2",
-    "typedoc": "^0.26.11",
-    "typescript": "^5.6.3",
+    "react": "^19.0.0",
```

Comment:
Why do we have the react dependency here?

Reply:
tbh I don't know, I thought it was intentional 🤔  I'll remove it and rebuild deps